### DISCUSSION TITLE
Creating user scram credential

### DISCUSSION THREAD

**Discussion by IDWNTL** [segmentio/kafka-go#1343] [Q&A]
Is there any possibility to create user via kafka-go. 
That I want: https://www.pulumi.com/registry/packages/kafka/api-docs/userscramcredential/

**Comment by RebelRaider** [comment]
**I interested in it too**

**Comment by HusenHuang** [comment]
I interested in it too

**Comment by petedannemann** [comment]
Yes this is supported https://github.com/segmentio/kafka-go/blob/a558bb8629a8f5a920a8e1bb640064ad7efb7cd8/alteruserscramcredentials.go#L63
